docker-compose.yml

```diff
@@ -33,7 +33,7 @@ services:
         - app.env
         - db.env
     # there is no `depend_on: condition:` in compose service v3
-    entrypoint: sh -c 'sleep 60; /app/entrypoint.sh'
+    entrypoint: sh -c 'sleep 5; /app/entrypoint.sh'
```

Comment:
Same as below. Is this an accident or intent?

Reply:
I think neither of them, reasoning is in my previous comment.